src/main/resources/application.yaml

```diff
@@ -54,13 +54,12 @@ management:
       enabled: true
       probes:
         enabled: true
-      group.liveness.include: livenessState,diskSpace,refreshScope
-      group.readiness.include: readinessState
+      diskspace.enabled: true
   endpoints:
     web:
       exposure:
-        include: health, metrics, prometheus
-  metrics.export.prometheus.enabled: true
+        include: health, metrics, prometheus, refresh
+  prometheus.metrics.export.enabled: true
```

Comment:
Was giving me some QQ about the Actuator management properties and boy were they not well maintained. With the changes, I believe many of our other repositories are also misconfigured by now and should probably replicate these changes in more of them.

Reply:
ah you found this before anyone else. this change in SB 3.1.0 is a pain